src/common/options.cc

```diff
@@ -4515,7 +4515,11 @@ std::vector<Option> get_global_options() {
 
     Option("bluestore_rocksdb_options", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("compression=kNoCompression,max_write_buffer_number=4,min_write_buffer_number_to_merge=1,recycle_log_file_num=4,write_buffer_size=268435456,writable_file_max_buffer_size=0,compaction_readahead_size=2097152,max_background_compactions=2,max_total_wal_size=1073741824")
-    .set_description("Rocksdb options"),
+    .set_description("Full set of rocksdb settings to override"),
+
+    Option("bluestore_rocksdb_options_annex", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("")
+    .set_description("An addition to bluestore_rocksdb_options. Allows to extend it without the need to provide all the set"),
```

Comment:
```suggestion
    .set_description("An addition to bluestore_rocksdb_options. Allows setting rocksdb options without repeating the existing defaults."),
```

Reply:
should add this to the release notes and docs as well

Reply:
Done for release notes.
But I don't see any relevant location in docs - looks like we don't cover most of bluestore settings there including the parent bluestore_rocksdb_options parameter. 
@jdurgin - do you have any suggestions?

Reply:
could add a new section in the bluestore config reference: https://docs.ceph.com/en/latest/rados/configuration/bluestore-config-ref/ - maybe call it 'advanced' or something since most users don't need to mess with rocksdb settings?